src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectWithParameterizedConstructorConverter.cs

```diff
@@ -279,7 +279,9 @@ internal sealed override bool OnTryRead(ref Utf8JsonReader reader, Type typeToCo
         /// <summary>
         /// Performs a full first pass of the JSON input and deserializes the ctor args.
         /// </summary>
+#if !DEBUG
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+#endif
```

Comment:
Why are these ifdefs needed? Inlining doesn't happen in debug, anyway.

Reply:
It doesn't? I've never been able to hit breakpoints in these methods.

Reply:
@EgorBo, am I wrong?

Reply:
Yes, it shouldn't happen in Debug